posts/2022/esds-fall-event.md

```diff
@@ -0,0 +1,37 @@
+---
+author: Julia Kent
+date: 2022-10-10
+tags: tutorial hackathon events
+---
+
+# A (Re)Introduction to Earth System Data Science (ESDS) Across NCAR/UCAR/UCP
+
+ESDS is hosting a (Re)Introduction to Earth System Data Science (ESDS) Across NCAR/UCAR/UCP on **November 9th and 10th** (Wednesday-Thursday). This event will be hybrid, in-person in NCAR's Mesa Lab Main Seminar Room and virtual over Zoom.
+
+The ESDS Fall Event will cover a (re)introduction to ESDS and Python tutorials (open to the community). Particular emphasis will be placed on the skills needed to best leverage the GeoCAT Python stack. The event will culminate in a hackathon (NCAR/UCAR personnel only) at the end of the second day.
+
+Here is an approximate content schedule:
+
+Day 1:
+
+- Intro to ESDS
+- Python Intro and Workspace Set Up
+- Python Syntax Basics\*
+- Numpy\*
+- Data Visualization\*
+- Xarray\*
+- Dask\*
+
+Day 2:
+
+- Intro to GeoCAT
+- GeoCAT-viz
+- GeoCAT-comp
+- UXarray
+- WRF-Python
+- GeoCAT on NCAR's HPC clusters
+- Hackathon (UCAR/NCAR only)
```

Comment:
Maybe add into Day 2:

- Opening (or something like that, which could introduce a few hack ideas and form groups based on people's interests?)